src/main/java/duke/tasks/Deadline.java

```diff
@@ -0,0 +1,30 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    protected LocalDate by;
+    protected String byDate;
+
+    public Deadline(String description, String by) {
+        super(description);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        this.by = LocalDate.parse(by, formatter);
+        DateTimeFormatter byFormatter = DateTimeFormatter.ofPattern("MMMM dd yyyy");
+        this.byDate = this.by.format(byFormatter);
+    }
+
+    public Deadline(String description, String by, Boolean other) {
```

Comment:
A suggestion - You could rename this variable to follow the naming guidelines like the other variables you have named

Reply:
Isn't the variable in camel case?